src/command_modules/azure-cli-container/HISTORY.rst

```diff
@@ -3,6 +3,11 @@
 Release History
 ===============
 
+0.3.6
++++++
+* Allow using only subnet ID to setup a virtual network for the container group
+* Add '--vnet-resource-group' to allow creating vnet and subnet in seperate resource group from the container group
```

Comment:
No. This goes against CLI command authoring guidelines.
https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#supporting-name-or-id-parameters